client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/steps/use-steps.tsx

```diff
@@ -0,0 +1,132 @@
+import { useTranslate } from 'i18n-calypso';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { MaybeLink } from './maybe-link';
+import type { Task, Expandable } from '@automattic/launchpad';
+
+const removeDuplicatedSlashes = ( url: string ) => url.replace( /(https?:\/\/)|(\/)+/g, '$1$2' );
+
+const getPluginInstallationPage = ( fromUrl: string ) => {
+	if ( fromUrl !== '' ) {
+		return removeDuplicatedSlashes(
+			`${ fromUrl }/wp-admin/plugin-install.php?s=%2522migrate%2520guru%2522&tab=search&type=term`
+		);
+	}
+
+	return 'https://wordpress.org/plugins/migrate-guru/';
+};
+
+const getMigrateGuruPageURL = ( siteURL: string ) =>
+	removeDuplicatedSlashes( `${ siteURL }/wp-admin/admin.php?page=migrateguru` );
+
+const recordInstructionsLinkClick = ( linkname: string ) => {
+	recordTracksEvent( 'calypso_site_migration_instructions_link_click', {
+		linkname,
+	} );
+};
+
+interface Options {
+	fromUrl: string;
+}
+
+interface Step {
+	task: Task;
+	expandable?: Expandable;
+}
+
+type Steps = Step[];
+
+export const useSteps = ( { fromUrl }: Options ): Steps => {
+	const translate = useTranslate();
+
+	const steps = [
+		{
+			title: translate( 'Install the Migrate Guru plugin' ),
+			content: (
+				<p>
+					{ translate(
+						'First you’ll need to install and activate the {{a}}Migrate Guru plugin{{/a}} on the site you want to migrate. Click next when you’re ready.',
+						{
+							components: {
+								a: (
+									<a
+										href={ getPluginInstallationPage( fromUrl ) }
+										target="_blank"
+										rel="noreferrer noopener"
+										onClick={ () => recordInstructionsLinkClick( 'install-plugin' ) }
+									/>
+								),
+							},
+						}
+					) }
+				</p>
+			),
+		},
+		{
+			title: translate( 'Get your site ready' ),
+			content: (
+				<>
+					<p>
+						{ translate(
+							'Head to the {{a}}Migrate Guru page on the site you’re migrating{{/a}}, Tap in your email address and click {{strong}}‘%(migrateLabel)s’{{/strong}}.',
```

Comment:
And here there is a comma instead of a period. Also "Tap in" sounds weird. "Enter" sounds better.

If this hasn't been through a copy review, we should probably do that now.

Reply:
Your suggestion was already applied as part of the copy review. https://github.com/Automattic/wp-calypso/pull/92070#discussion_r1672966045